packages/@headlessui-react/src/utils/render.ts

```diff
@@ -11,7 +11,7 @@ import {
   type ReactElement,
   type Ref,
 } from 'react'
-import type { Expand, Props, XOR, __ } from '../types'
+import type { Expand, Props } from '../types'
```

Comment:
I believe `XOR` was only used here, so we can probably get rid of it. Same goes for the `__` type.

Reply:
Done